helpers/cleanup.sh

```diff
@@ -41,9 +42,10 @@ ORG_ID=""
 SERVICE_ACCOUNT_NAME=""
 WITH_ENFORCER=""
 HOST_PROJECT_ID=""
+ON_GKE=""
 
 OPTIND=1
-while getopts ":hef:p:o:s:" opt; do
+while getopts ":hekf:p:o:s:" opt; do
```

Comment:
Slight nit, but would be nice if the getopts order was the same as in `setup.sh`.

In `cleanup.sh`, it is: `":hekf:p:o:s:"`
In `setup.sh`, it is: `":hekp:f:o:"`

Reply:
agreed